chainer/function_hook.py

```diff
@@ -76,15 +84,24 @@ class FunctionHook(object):
        as a thread local object. So, function hooks registered
        are different depending on threads.
 
+    If the hook is registered in this way, :meth:`~chainer.FunctionHook.added`
+    and :meth:`~chainer.FunctionHook.deleted` are given ``None`` as the
+    ``function`` argument.
+
     The other one is to register directly to
     :class:`~chainer.FunctionNode` object with
-    :meth:`~chainer.Function.add_hook` method.
+    :meth:`~chainer.FunctionNode.add_hook` method.
     Function hooks registered in this way can be removed by
-    :meth:`~chainer.Function.delete_hook` method.
+    :meth:`~chainer.FunctionNode.delete_hook` method.
     Contrary to former registration method, function hooks are registered
     only to the function which :meth:`~chainer.FunctionNode.add_hook`
     is called.
```

Comment:
It's also not related to this change, but I think it's a good chance to fix some expressions:
- The other one is to `register directly` to... -> `register it directly`
- Contrary to `former registration method`, ... -> `the former registration method`
- only to the function which add_hook is called. -> only to the function whose add_hook method is called.